src/common/obj_bencher.h

```diff
@@ -64,7 +64,7 @@ typedef std::pair<std::string, std::string> Object;
 class ObjBencher {
   bool show_time;
   Formatter *formatter = NULL;
-  ostream *outstream = NULL;
+  std::ostream *outstream = NULL;
```

Comment:
nullptr (if already touching this line)

Reply:
i'd minimize the change. please note this changeset touches hundreds of files. i'd prefer to be focused.